github/pulls_reviewers.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// RequestReviewers submits a set of logins to be potential reviewers on a PR.
```

Comment:
I think it'd be nicer to stay closer to the language the GitHub API docs use and say that this method creates a "review request". What do you think of what I wrote earlier?

```
// RequestReviewers creates a review request for the provided GitHub users for the specified pull request.
```

That seems more consistent with the other endpoints (like the remove one).